gluten-core/src/main/scala/io/glutenproject/execution/GlutenWholeStageColumnarRDD.scala

```diff
@@ -20,6 +20,7 @@ import io.glutenproject.GlutenConfig
 import io.glutenproject.backendsapi.BackendsApiManager
 import io.glutenproject.metrics.IMetrics
 import io.glutenproject.substrait.plan.PlanBuilder
+
```

Comment:
these style changes are not related to this pr, but just run `mvn spotless:apply -Pbackends-velox -Pspark-3.3`..